alm/src/utils/contract.ts

```diff
@@ -59,40 +68,43 @@ export const getRequiredSignatures = async (
   return parseInt(requiredSignatures)
 }
 
-export const getValidatorList = async (contract: Contract, blockNumber: number, snapshotProvider: SnapshotProvider) => {
+export const getValidatorList = async (
+  contract: Contract,
+  blockNumber: number,
+  fromHome: boolean,
```

Comment:
can you rename fromHome to `isHome`. seems more easy to follow

Reply:
https://github.com/fuseio/tokenbridge/pull/7/commits/7ea6cb3b3ccaed3a7cb6f5f2e0b2db1268875c51
done